### PR TITLE
Seeding Script

### PR DESCRIPTION
**Pull request by phucciTr**

### CODE REVIEW

File: server/seeding-script/rawData/dataType/blankets.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+https://www.ikea.com/us/en/images/products/koelax-bedspread-dark-green__0723091_pe733853_s5.jpg?f=m 600w,
+https://www.ikea.com/us/en/images/products/koelax-bedspread-dark-green__0723091_pe733853_s5.jpg?f=s 500w,
+https://www.ikea.com/us/en/images/products/koelax-bedspread-dark-green__0723091_pe733853_s5.jpg?f=xs 400w,
+https://www.ikea.com/us/en/images/products/koelax-bedspread-dark-green__0723091_pe733853_s5.jpg?f=xxs 300w,
+https://www.ikea.com/us/en/images/products/koelax-bedspread-dark-green__0723091_pe733853_s5.jpg?f=xxxs 160w,
+https://www.ikea.com/us/en/images/products/koelax-bedspread-dark-green__0723091_pe733853_s5.jpg?f=u 80w,
+https://www.ikea.com/us/en/images/products/koelax-bedspread-dark-green__0723091_pe733853_s5.jpg?f=xu 40w,
+https://www.ikea.com/us/en/images/products/koelax-bedspread-dark-green__0723091_pe733853_s5.jpg?f=xxu 39w
+
+Ikea:10
```
Comment (joshparkerj) [inline server/seeding-script/rawData/dataType/blankets.js:17]: What does Ikea:10 mean here?

**Comment by joshparkerj** [comment]
Looks as good as long as these images from ikea will be put on s3

**Comment by phucciTr** [comment]
> Looks as good as long as these images from ikea will be put on s3

Thanks for the review Josh

File: server/seeding-script/rawData/dataType/blankets.js (hunk 1 above)
Comment (phucciTr) [reply server/seeding-script/rawData/dataType/blankets.js:17]: I did a query to get all images urls on Ikea and Ikea:10 just happened to be there in the query

**Pull request merged**